mappings/net/minecraft/test/StartupParameter.mapping

```diff
@@ -1,4 +1,258 @@
 CLASS net/minecraft/class_4516 net/minecraft/test/StartupParameter
 	FIELD field_20558 test Lnet/minecraft/class_4517;
```

Comment:
Hmm, maybe `testState` could be better since the class is `GameTestState` now? `test` is also fine by me though.

Reply:
I thought about that too, but `test` takes a lot less space. It could also be renamed `state`.